firefox-ios/Client/Frontend/Settings/AppearanceSettings/Generic views/GenericSectionView.swift

```diff
@@ -61,6 +67,8 @@ struct GenericSectionView<Content: View>: View {
             }
         }
         .padding(.bottom, UX.sectionPadding)
+        .accessibilityElement(children: .contain)
```

Comment:
@yoanarios this was the main bit really that caused failures since without it the button appeared with the accessibility identifier of the parent.